blocks/donor-wall/class-give-donor-wall.php

```diff
@@ -226,7 +226,7 @@ public function render_block( $attributes ) {
 			'readmore_text'     => $attributes['readMoreText'],
 			'loadmore_text'     => $attributes['loadMoreText'],
             'toggle_options'    => $attributes['toggleOptions'],
-            'filter'            => $attributes['filter'],
+            'filter_options'    => $attributes['filterOptions'],
```

Comment:
@JoshuaHungDinh you do not need to pass  `filter_options ` and  `toggle_options ` in attributes to render donor wall shortcode. Can we remove them from here?

You can keep them in the block attributes list.

Reply:
@ravinderk  Got it, I see what you mean. I made the same updates to the Form Grid as well.